src/libcore/lib.rs

```diff
@@ -199,6 +200,13 @@ pub mod iter;
 pub mod option;
 pub mod raw;
 pub mod result;
+#[unstable(feature = "c_variadic",
+           reason = "the `c_variadic` feature has not been properly tested on \
+                     all supported platforms",
+           issue = "27745")]
+#[cfg(any(target_arch = "aarch64", target_arch = "mips", target_arch = "powerpc",
+          target_arch = "powerpc64", target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
All of these architectures have a `va_list` implementation in the module. We could alternatively include the module regardless of the architecture but have a default implementation that does nothing or just calls `bug!`.

Reply:
This approach seems reasonable to me.

Reply:
Also seems fine to me, althought maybe each architecture could be written on one line, to make this `cfg` more obvious.